### ISSUE TITLE
Request textDocument/hover failed.

### ISSUE THREAD

**Issue by Albert-Gao** [relay-tools/vscode-apollo-relay#31]
Directives like `@argumentDefinitions` won't produce error anymore, thanks!

However, when hovering over `@argumentDefinitions`, it will trigger an Apollo VSC plugin error like this picture:

![image](https://user-images.githubusercontent.com/18282328/93541413-80ddd680-f9aa-11ea-85b4-a74afe015945.png)

Am I doing anything wrong, is this expected? I am totally fine with it, just want to make sure I do not miss anything

**Comment by alloy** [comment]
Thanks for the report, looks like this is caused because of how the argument definition parameters aren't actually static like with normal directives, so presumably the extension is trying to get metadata for amnestying argument that doesn’t exist. Will time-box some time to see if that’s easy to work around.

**Comment by alloy** [comment]
Yeah that’s what happens, it occurs [here](https://github.com/apollographql/apollo-tooling/blob/cfc121bda7c9f68d22203381b14e1d11c4935aff/packages/apollo-language-server/src/languageProvider.ts#L378-L394) in the Apollo language server. Other than perhaps monkey-patching `typeInfo` at runtime, I don’t see any existing opportunities to making it work.

Going to pass on this for now, but I’ll leave the issue open for if/when somebody wants to pick it up.
